### PR TITLE
Add test for BZ 1943306

### PR DESCRIPTION
**Pull request by sambible**
Simple test making sure that the "Days From Now" field in the template has a default set.

Testing BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1943306

### CODE REVIEW

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_reporttemplates.py -k 'test_entitlements_report_no_inputs_field'

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_reporttemplates.py -k 'test_entitlements_report_no_inputs_field'

**Review by ColeHiggins2 — APPROVED** [review]

File: tests/foreman/cli/test_reporttemplates.py [hunk 1]
```diff
@@ -857,6 +857,40 @@ def test_positive_schedule_entitlements_report(
...
+
+    :expectedresults: report is generated, and the Days From Now field isn't required
+
+    :BZ: 1943306
+
+    :customerscenario: true
+    """
+    client = rhel7_contenthost
+    client.install_katello_ca(target_sat)
+    client.register_contenthost(module_entitlement_manifest_org.label, local_ak['name'])
```
Comment (vsedmik) [inline tests/foreman/cli/test_reporttemplates.py:881]:
```suggestion
    client.register(
        module_entitlement_manifest_org, module_location, local_ak['name'], target_sat
    )
```
Via global registration.

**Review by vsedmik — COMMENTED** [review]
One proposal to consider.

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_reporttemplates.py -k 'test_entitlements_report_no_inputs_field'

**Review by vsedmik — APPROVED** [review]

**Pull request merged**